### PR TITLE
Add a file for Virginia, MN

### PR DESCRIPTION
**Pull request by andrew-oberstar**
The purpose of this is to add a hometown file for Virginia, MN.

Some :question: that I want you all to help me answer.

1. If you've been to Virginia, MN what did you do? What did you think of it?
1. If you have kids, what kind of things do they like to do at night?
1. What kind of food do you like?

### CODE REVIEW

File: andrew-oberstar-virginia-mn.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Virginia, MN
+
+### Restaurants
+
+
+
+### Places the locals like
+
+What places?
```
Comment (joshuadruley) [inline andrew-oberstar-virginia-mn.md:9]: Where can I find a good cup of coffee?

File: andrew-oberstar-virginia-mn.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Virginia, MN
+
```
Comment (joshuadruley) [inline andrew-oberstar-virginia-mn.md:2]: Love to find a cabin in Virginia.  Where would I start looking?

File: andrew-oberstar-virginia-mn.md (hunk 1 above)
Comment (andrew-oberstar) [reply andrew-oberstar-virginia-mn.md:9]: Maybe the [Village Inn](http://www.villageinn.com/) or the Starbuck's in the Target.

File: andrew-oberstar-virginia-mn.md (hunk 2 above)
Comment (andrew-oberstar) [reply andrew-oberstar-virginia-mn.md:2]: Wouldn't have a cabin *in* Virginia, but Lake Vermillion is close (20 miles).

**Comment by AdamPaolo** [comment]
Maybe @vpierson can weigh in...

File: andrew-oberstar-virginia-mn.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+### Restaurants
+
```
Comment (joshuadruley) [inline andrew-oberstar-virginia-mn.md:4]: Village Inn sounds amazing -- pancakes :smiley:

**Review by joshuadruley — APPROVED** [review]
Looks awesome :+1: yeah :shipit:

**Pull request merged**